README.md

```diff
@@ -61,7 +61,10 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
-      timeout_seconds:  1
+      timeout_seconds:  1,
+
+      # lambda determining what response is considered a failure 
+      error_response: lambda { |response| response.status >= 500 }
```

Comment:
That is actually not correct, the error_response is set on the middleware not the circuit.